desc/magnetic_fields/_core.py

```diff
@@ -1103,6 +1231,10 @@ def compute_magnetic_field(
         transforms : dict of Transform
             Transforms for R, Z, lambda, etc. Default is to build from source_grid
             Unused by this MagneticField class.
+        chunk_size : int or None
+            Size to split computation into chunks of evaluation points.
+            If no chunking should be done or the chunk size is the full input
+            then supply ``None``. Default is ``None``.
```

Comment:
to be extra annoying , `"Unused by this MagneticField class"` would be good to add for this for the fields that dont actually use it like TF etc

Reply:
The current API for all coils and magnetic fields would benefit from hiding or documenting unused parameters. `chunk_size` is just one of three such parameters. For example, https://desc-docs--1568.org.readthedocs.build/en/1568/_api/magnetic_fields/ToroidalMagneticField/desc.magnetic_fields.ToroidalMagneticField.compute_Bnormal.html#desc.magnetic_fields.ToroidalMagneticField.compute_Bnormal. A separate PR can be made to make this change for all parameters in a unified way.

Reply:
done